articles/cross-origin-authentication/fingerprinting.md

```diff
@@ -10,7 +10,7 @@ useCase:
 ---
 # Fingerprinting of Username + Password Login Requests
 
-Auth0 has announced a [vulnerability](https://auth0.com/blog/managing-and-mitigating-security-vulnerabilities-at-auth0/) in all versions of Lock (<11) and auth0.js (<9). We strongly recommend that all users of legacy versions migrate to the newest, most secure versions immediately. Support for conducting logins with a username and password with these versions is now end of life and deprecated versions will stop working on **July 16th, 2018**.
+Auth0 has announced a [vulnerability](https://auth0.com/blog/managing-and-mitigating-security-vulnerabilities-at-auth0/) in all versions of Lock (<11) and auth0.js (<9). We strongly recommend that all users of legacy versions migrate to the newest, most secure versions immediately. Support for conducting logins with a username and password with these versions is now deprecated.
```

Comment:
```suggestion
Auth0 has announced a [vulnerability](https://auth0.com/blog/managing-and-mitigating-security-vulnerabilities-at-auth0/) in all versions of Lock (<11) and auth0.js (<9). We strongly recommend that all users of legacy versions migrate to the newest, most secure versions immediately. Support for conducting logins with a username and password with these versions is no longer supported, and in most cases, not possible.
```

Reply:
(Deprecated is the wrong word to use since the endpoints are already end-of-life.)